pkg/config/loader.go

```diff
@@ -147,6 +149,39 @@ func LoadClient(cfg *Config) (kubernetes.Interface, error) {
 	return clientset, nil
 }
 
+func OutOfClusterClient() (kubernetes.Interface, error) {
```

Comment:
This should probably have a comment. What does `OutOfClusterClient` mean?

Reply:
I added a comment, the language came from https://github.com/kubernetes/client-go/tree/master/examples 

in cluster is stuff running in the cluster, like sonobuoy https://github.com/heptio/sonobuoy/blob/master/pkg/config/loader.go#L135

and similarly out of cluster is stuff accessing the cluster from outside

Reply:
that makes sense! I don't love that it doesn't take any arguments, though. Could it take a kubeconfig filename?